builder/bundles.go

```diff
@@ -480,7 +506,35 @@ func downloadRpms(packagerCmd, rpmList []string, baseDir string, maxRetries int)
 	return nil, downloadErr
 }
 
-func installBundleToFull(packagerCmd []string, buildVersionDir string, bundle *bundle, downloadRetries int) error {
+func extractRpm(baseDir string, rpm string) error {
+	cmd := exec.Command("rpm2cpio", rpm)
+	cmd.Env = os.Environ()
```

Comment:
Do you need to set env here?

Reply:
yes as it checks if rpm2cpio is installed or not